sunpy/net/helioviewer.py

```diff
@@ -23,11 +23,11 @@
 
 class HelioviewerClient(object):
     """Helioviewer.org Client"""
-    def __init__(self, url="https://legacy.helioviewer.org/api/"):
+    def __init__(self, url="https://api.helioviewer.org/"):
         """
         url : location of the Helioviewer API.  The default location points to
-            version 1 of the API.  Version 1 of the Helioviewer API is
-            currently planned to be supported until the end of April 2017.
+            version 2 of the API.  Version 2 of the Helioviewer API is
+            better than the previous version and provides more features.
```

Comment:
Not sure we need this sentence.

Reply:
It does provide more features

Reply:
Sure but I feel its redundant. We support v2 because its the current API and who knows when the old one disappears.

Reply:
Ok I will remove it